### DISCUSSION TITLE
I am getting  Failed to create new item - {"0":{"title":"database is locked"}}

### DISCUSSION THREAD

**Discussion by kawpii** [daptin/daptin#128] [Q&A] [answered]
I am getting following error when trying to create a table

`Failed to create new item - {"0":{"title":"database is locked"}}`

Why is that? I am just using default SQLite database.

---------------------

Something else I noticed, deleted items appear until you refresh or navigate to some other page and navigate back to the previous page

**Comment by artpar** [comment, marked as answer]
I did notice this myself as well last week. Seems specific to SQLite. Needs to be identified fixed.

Created an issue for this: https://github.com/daptin/daptin/issues/129

**Reply by kawpii** [reply]
Thanks for the quick response!

What about
>  deleted items appear until you refresh or navigate to some other page and navigate back to the previous page

**Reply by artpar** [reply]
So that is related to the UI Dadadash https://github.com/daptin/dadadash

I do remember having handling deletion and removal of item from the UI. 

Just to be clear you are talking about items inside a base (for example deleting a calendar ?)

**Reply by kawpii** [reply]
> Just to be clear you are talking about items inside a base (for example deleting a calendar ?)
I observed this behavior with tables
